### PR TITLE
RD-3010 Fixed error display, change modal header title

### PR DESCRIPTION
**Pull request by kubama**
## Description
See https://cloudifysource.atlassian.net/browse/RD-3049 and https://cloudifysource.atlassian.net/browse/RD-3051

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires a documentation update.
- [X] I added proper labels to this PR.

## Tests
https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/1206/ - only 'getting started' spec

## Documentation
N/A

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "gettingStartedModal": {
         "disableModalLabel": "Don't show next time",
         "titles": {
-            "technologiesStep": "Getting Started",
+            "technologiesStep": "First, please select your environment(s)",
```
Comment (qooban) [inline app/translations/en.json:31]: I think there's one place in `getting_started_spec.ts` which should be updated after that change https://github.com/cloudify-cosmo/cloudify-stage/blob/9229fd38f4dc948b2892fcc0ca6d48d35aba5351/test/cypress/integration/getting_started_spec.ts#L329

File: app/translations/en.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
             "blueprintsMetadataLoadingError": "Blueprints information loading error"
         },
         "validation": {
-            "someTechnologyRequiredError": "At least one technology needs to be selected.",
+            "someTechnologyRequiredError": "At least one environment needs to be selected.",
```
Comment (qooban) [inline app/translations/en.json:48]: As we are changing the labels from "technology" to "environment" shouldn't we also change internal names (variables, translation keys, etc.)?

**Review by qooban — COMMENTED** [review]
Commented.

File: app/translations/en.json (hunk 2 above)
Comment (kubama) [reply app/translations/en.json]: Done

File: app/translations/en.json (hunk 1 above)
Comment (kubama) [reply app/translations/en.json]: Updated

File: app/components/GettingStartedModal/formValidation.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-import type { GettingStartedSchemaSecret, GettingStartedSecretsData, GettingStartedTechnologiesData } from './model';
+import type { GettingStartedSchemaSecret, GettingStartedSecretsData, GettingStartedEnvironmentsData } from './model';
 
 const t = StageUtils.getT('gettingStartedModal.validation');
 
-export const validateTechnologyFields = (data: GettingStartedTechnologiesData) => {
+export const validateEnvironmentsFields = (data: GettingStartedEnvironmentsData) => {
     if (!_.some(data)) {
-        return t('someTechnologyRequiredError');
+        return t('environmentRequiredError');
```
Comment (qooban) [inline app/components/GettingStartedModal/formValidation.ts:9]: Good decision 👍🏻

File: app/components/GettingStartedModal/formValidation.ts [hunk 4]
```diff
@@ -1,12 +1,12 @@
 import StageUtils from '../../utils/stageUtils';
 
-import type { GettingStartedSchemaSecret, GettingStartedSecretsData, GettingStartedTechnologiesData } from './model';
+import type { GettingStartedSchemaSecret, GettingStartedSecretsData, GettingStartedEnvironmentsData } from './model';
 
 const t = StageUtils.getT('gettingStartedModal.validation');
 
-export const validateTechnologyFields = (data: GettingStartedTechnologiesData) => {
+export const validateEnvironmentsFields = (data: GettingStartedEnvironmentsData) => {
```
Comment (qooban) [inline app/components/GettingStartedModal/formValidation.ts:7]: Plural makes more sense here 👍🏻

**Review by qooban — APPROVED** [review]
Looks good! Thanks for spending extra-time on naming alignments 👍🏻

**Comment by kubama** [comment]
@stepek Waiting for your review 🙂

**Review by stepek — APPROVED** [review]

**Pull request merged**